components/css/buckyless/header.less

```diff
@@ -112,4 +112,23 @@ portal-header {
   .header__bell {
     display: inline-block;
   }
+
+  .header__guest-login-btn {
+    font-weight: 600;
+    &:hover, &:focus {
```

Comment:
[linter flags style issues with this less](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679020979#L753).